src/Util/Temporal.php

```diff
@@ -317,7 +317,7 @@ public static function getRelativeDate(string $posted_date, string $format = nul
 
 		$abs = strtotime($localtime);
 
-		if (is_null($posted_date) || $posted_date <= DBA::NULL_DATETIME || $abs === false) {
+		if (empty($posted_date) || $posted_date <= DBA::NULL_DATETIME || $abs === false) {
 			return DI::l10n()->t('never');
 		}
```

Comment:
Now that I have had a better look at it, it looks like the null value is warranted, it should return "Never" before the string concatenation and the `strtotime()` call like this:
```suggestion
		if (empty($posted_date) || $posted_date <= DBA::NULL_DATETIME) {
			return DI::l10n()->t('never');
		}
		
		$localtime = $posted_date . ' UTC';
		$abs = strtotime($localtime);
		if ($abs === false) {
			return DI::l10n()->t('never');
		}
```

Reply:
Okay, let's take a closer look. Some table columns indeed have `null` as value, e.g. "`updated`" or "`edited`" columns where on creation of a new record, they are `null` (`DATETIME NOT NULL`). So accepting `null` again makes sense.  But when I look at the method name and parameters I would expect a "please get (read: give) me a relative date from `$post_date`!" and then when I invoke it with `Temporal::getRelativeDate()` it means "please give me a relative date from nothing". Okay, that's "never" again. Hmm.

Reply:
Plus when `strtotime()` is returning `false` we interpret it as "never" while the documentation says error/bogus. Like e.g. `$bogus = strtotime('Not good');` will return `false`.